src/utils/getNormalizedOptions.ts

```diff
@@ -28,7 +47,8 @@ export function getNormalizedOptions(this: loader.LoaderContext): StyleResources
 
     if (typeof injector !== 'function') {
```

Comment:
Could you add more assertion here? Such as !(typeof injector === 'function' || Object.keys(specialInjectors).includes(injector))